Client/Frontend/Reader/ReaderMode.swift

```diff
@@ -36,10 +36,48 @@ enum ReaderModeFontType: String {
 
 enum ReaderModeFontSize: Int {
     case Smallest = 1
-    case Small
+    case Small = 2
     case Normal = 3
     case Large = 4
     case Largest = 5
+
+    func isSmallest() -> Bool {
+        return self == Smallest
+    }
+
+    func smaller() -> ReaderModeFontSize {
+        switch self {
+        case Smallest:
```

Comment:
I know this is not ideal but at least there is very little that can go wrong this way :-)